views/index.html

```diff
@@ -18,27 +18,27 @@
           </div>
           <h3>Product Development</h3>
             <p>
-              We build <i>mobile-first</i> MVPs with intuitive and user-friendly interfaces.
+              We build responsive, mobile-first MVPs with intuitive and human-centred design.
             </p>
         </div>
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon user-testing-icon">
           </div>
-          <h3>User testing</h3>
+          <h3>Co-working</h3>
```

Comment:
@iteles I know it is - I just thought @nelsonic didn't like using the word user - I remember him mentioning it in a meeting